examples/environment.yml

```diff
@@ -5,6 +5,12 @@ dependencies:
   - intake
   - jupyter
   - nodejs
+  - scipy
+  - scikit-image
+  - s3fs
+  - tqdm
+  - intake::intake
+  - intake::intake-xarray
```

Comment:
intake and intake-xarray don't need to come from the intake channel - conda-forge is fine.